overview/saml.mdx

```diff
@@ -15,6 +15,51 @@ which users have access to your product. They could give everyone at the company
 access, specify teams like only the engineering team, or even pick specific
 people to give access.
 
+### SAML Setup Guide
+
+We're going to set up our PropelAuth Project to enable Acme Co to set up a SAML connection.
+
+1. On the dashboard for your project, click the "View Org Page" button under the Test Environment. When you click this, you are now viewing the organization management page for your product. In other words, you are now acting as if you were an employee at Acme Co. If you haven't done it yet, you'll need to sign up before you can view the org page.
+
+![dashboard](/images/saml-setup-3.png)
+
+2. Click the Create Organization button.
+
+![create organization](/images/saml-setup-4.png)
+
+3. Write a name for the organization, and click Create Organization.
+
+![organization game](/images/saml-setup-9.png)
+
+4. Back in the project dashboard, under the Management section of your Project, select the Organizations tab, and click the edit icon for the organization in which you would like to enable SAML for.
+
+![organization settings](/images/saml-setup-11.png)
+
+5. On this page, click the Edit button on the top right.
+
+![edit organization](/images/saml-setup-12.png)
+
+6. Toggle the Can Enable SAML option, and click Update.
+
+![enable saml](/images/saml-setup-13.png)
+
+7. Back on the page for the new Organization, click the Settings button, and click Enable SAML.
```

Comment:
Tiny edit: I'd add something like "Now we can go back to being a user at Acme Co" - just to make the transition smoother